packages/cli/src/evaluation.ee/test-runner/test-runner.service.ee.ts

```diff
@@ -313,6 +301,34 @@ export class TestRunnerService {
 			},
 		};
 
+		if (
+			!(
+				config.get('executions.mode') === 'queue' &&
+				process.env.OFFLOAD_MANUAL_EXECUTIONS_TO_WORKERS === 'true'
+			) &&
+			data.executionData
+		) {
+			const nodeExecutionStack: IExecuteData[] = [];
+			nodeExecutionStack.push({
+				node: triggerNode,
+				data: {
+					main: [[{ json: {} }]],
+				},
+				source: null,
+			});
+
+			data.executionData.executionData = {
+				contextData: {},
+				metadata: {},
+				// workflow does not evaluate correctly if this is passed
+				// in queue mode with offload manual executions
+				// but this is expected otherwise and will fail if not passed
```

Comment:
For future reference, we're taking some debt here - eventually we'll need to rethink this specific change, to unblock unifying both queue mode branches at n8n v2.

Reply:
Sadly no. I'd expect this year.